Ray.jl/test/function_manager.jl

```diff
@@ -0,0 +1,84 @@
+@testset "function manager" begin
+    using Ray: FunctionManager, export_function!, import_function!, wait_for_function
+    using ray_core_worker_julia_jll: JuliaGcsClient, Connect, function_descriptor, JuliaFunctionDescriptor, Exists
+
+    client = JuliaGcsClient("127.0.0.1:6379")
+    Connect(client)
+
+    fm = FunctionManager(client, Dict{String,Any}())
+
+    jobid = 1337
+
+    f = x -> isless(x, 5)
+    export_function!(fm, f, jobid)
+
+    fd = function_descriptor(f)
```

Comment:
I'd find it helpful to know the functionality being tested if we organised these into`@testsets`